nimble/controller/include/controller/ble_ll.h

```diff
@@ -34,6 +35,14 @@ extern "C" {
 #error 32.768kHz clock required
 #endif
 
+#define BLE_LL_ASSERT(cond) \
+    if (!(cond)) { \
```

Comment:
this should be enabled only for debug builds

Reply:
Well I had such version but then got to conclusion that it would be nice to always know if assert happens. However in order to keep current behavior I will enable it only for debug.

Reply:
leaving this always enabled means extra code and branching included in build